src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/CreateSDOCallbackHandler.java

```diff
@@ -604,6 +605,23 @@ private CallbackResponse prePopulateOrderDetailsPages(CallbackParams callbackPar
 
         updatedData.smallClaimsWitnessStatement(tempSmallClaimsWitnessStatement).build();
 
+        SmallClaimsFlightDelay tempSmallClaimsFlightDelay = SmallClaimsFlightDelay.builder()
```

Comment:
Add changes behind the featureToggle.

Reply:
Have added a condition with R2 toggle as suggested.